.gitignore

```diff
@@ -1,3 +1,5 @@
+/.vscode
```

Comment:
The `.vscode` folder is ignored so one has to do

```bash
git add -f .vscode/settings.json
```

if he wants to commit the contents.